src/ai/backend/manager/models/vfolder.py

```diff
@@ -734,6 +739,53 @@ async def filter_host_allowed_permission(
     return allowed_hosts
 
 
+async def delete_vfolder_by_ids(
+    db_session: SASession,
+    storage_manager: StorageSessionManager,
+    storage_ptask_group: aiotools.PersistentTaskGroup,
+    *,
+    vfolder_infos: Sequence[tuple[uuid.UUID, str]],  # id, host
```

Comment:
Using `tuple` to represent complex data type like this can make code reader hard to understand the context. Please rewrite this to use `TypedDict` or `NamedTuple`.

Reply:
Updated. thank you!